docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -17,63 +29,55 @@ Distributing collections involves three major steps:
 
 .. _config_distribution_server:
 
-Configuring your distribution server or servers
-================================================
+Initial configuration of your distribution server or servers
+============================================================
+
+Configure a connection to one or more distribution servers so you can publish collections there. You only need to configure each distribution server once. You must repeat the other steps (building your collection tarball, preparing to publish, and publishing your collection) every time you publish a new collection or a new version of an existing collection.
 
-1. Get a namespace on each distribution server you want to use (Galaxy, private Automation Hub, Red Hat Automation Hub).
+1. Create a namespace on each distribution server you want to use.
 2. Get an API token for each distribution server you want to use.
 3. Specify the API token for each distribution server you want to use.
 
 .. _get_namespace:
 
-Getting a namespace
--------------------
+Creating a namespace
+--------------------
 
-You need a namespace on Galaxy and/or Automation Hub to upload your collection. To get a namespace:
+You must upload your collection into a namespace on each distribution server. If you have a login for Ansible Galaxy, your Galaxy username is also a Galaxy namespace. You can create additional namespaces on Ansible Galaxy if you choose. For Red Hat Automation Hub and private Automation Hub you must create a namespace before you can upload your collection. To create a namespace:
```

Comment:
> If you have a login for Ansible Galaxy, your Galaxy username is also a Galaxy namespace.

Unless you have hyphens in your username.

Reply:
Wow. Okay, that's a good catch. Thanks @s-hertel, I'll add that.

Reply:
Addressed in 60a5bfd89a256ebc871c5ecb87d7539c1c248c56. Thanks @s-hertel.